config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'splash#splashpage'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  devise_for :users
+
+  resources :categories do 
+    resources :expenses 
```

Comment:
- [ ] Please make sure that the specified routes are all used. This is for enhancing the project's optimization and also making it more secure. Currently, the only used ones are `index` and `new`. Please modify the resources using the `only` method. 🙏

Reply:
well resolved! ✔️

Reply:
@rloterh 
Thanks for your valuable time.